xcode/Subconscious/Shared/Components/OmniboxView.swift

```diff
@@ -20,23 +20,29 @@ struct OmniboxView: View {
             HStack(spacing: 0) {
                 Spacer()
                 Text(verbatim: title)
+                    .font(.callout)
                     .foregroundColor(.accentColor)
                 Spacer()
             }
         }
         .padding(.leading, 8)
         .padding(.trailing, 12)
-        .frame(height: 36)
-        .background(Color.secondaryBackground)
-        .cornerRadius(12)
+        .frame(height: 34)
```

Comment:
Visual tweak for better look-and-feel in navigation view title bar.

Reply:
More magic numbers, but these don't seem worth adding to `AppTheme` to me.

Reply:
@bfollington Yup. This is a unique case because Omnibox lives in a Navigation toolbar, which is fixed height. The magic number here is what looks visually right for this particular component in this particular context. I don't think its useful elsewhere.